sources/Cargo.lock

```diff
@@ -559,6 +559,91 @@ dependencies = [
  "serde_json",
 ]
 
+[[package]]
+name = "async-broadcast"
+version = "0.7.2"
```

Comment:
Can we disable the async feature of zbus to avoid bringing in these additional dependencies? Not sure what async is really buying us in this context.